src/Tribe/Event_Tickets/Attendees_Report.php

```diff
@@ -44,10 +44,20 @@ public function event_details_top( $event_id ) {
 		if ( tribe_has_venue( $event_id ) ) {
 			$venue_id = tribe_get_venue_id( $event_id );
 
+			/**
+			 * Allows for control of the specific "edit post" URLs used for venue.
+			 *
+			 * @since TBD
+			 *
+			 * @param string $link     The default "edit post" URL.
+			 * @param int    $venue_id The Post ID of the venue.
+			 */
+			$edit_venue_link = apply_filters( 'tribe_tickets_venue_action_links_edit_url', get_edit_post_link( $venue_id ), $venue_id );
+
 			echo '
 				<li class="venue-name">
 					<strong>' . tribe_get_venue_label_singular() . ': </strong>
-					<a href="' . get_edit_post_link( $venue_id ) . '" title="' . esc_html__( 'Edit Venue', 'the-events-calendar' ) . '">' . tribe_get_venue( $event_id ) . '</a>
+					<a href="' . esc_url( $edit_venue_link ) . '" title="' . esc_html__( 'Edit Venue', 'the-events-calendar' ) . '">' . tribe_get_venue( $event_id ) . '</a>
```

Comment:
Do we need to escape tribe_get_venue( $event_id )? That is what I believe tribot is picking up

Reply:
It's because of the `echo`, no worries.